setup.py

```diff
@@ -201,6 +201,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'celery~=4.4.2',
     'flower>=0.7.3, <1.0',
     'tornado>=4.2.0, <6.0',  # Dep of flower. Pin to a version that works on Py3.5.2
+    'vine~=1.3',  # https://stackoverflow.com/questions/32757259/celery-no-module-named-five
```

Comment:
I read about it in your slack thread but this should be a separate PR imo.

Reply:
Agree

Reply:
Separated it out to https://github.com/apache/airflow/pull/10828 with more explanation